server/src/uuid/model/taxonomy_term/mod.rs

```diff
@@ -236,3 +242,120 @@ impl TaxonomyTerm {
         typename.to_case(Case::Camel)
     }
 }
+
+#[derive(Error, Debug)]
+pub enum TaxonomyTermSetNameAndDescriptionError {
+    #[error(
+        "Taxonomy Term name and description cannot be set because of a database error: {inner:?}."
+    )]
+    DatabaseError { inner: sqlx::Error },
+    #[error(
+        "Taxonomy Term name and description cannot be set because of an event error: {inner:?}."
+    )]
+    EventError { inner: EventError },
+}
```

Comment:
```suggestion
```

You should be able to only use `operation::Error`

Reply:
done in 3f20e00